### PR TITLE
[#1929] Improvement: fix method annotation

### PR DESCRIPTION
**Pull request by YxAc**
### What changes were proposed in this pull request?
- fix method annotation

### Why are the changes needed?
Fix: #1929 

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
No

### CODE REVIEW

**Comment by YxAc** [comment]
@qqqttt123 I have split this PR from #2149 , please help review, thx.

File: common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java [hunk 1]
```diff
@@ -212,7 +212,7 @@ String convert(JsonNode element) {
     }
   }
 
-  private static volatile ObjectMapper mapper = null;
```
Comment (qqqttt123) [inline common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java:215]: This is correct without issues. But I prefer using lazy Singleton like https://blog.csdn.net/iteye_20954/article/details/81342188 . I feel that it's more elegant. You can raise another pr to fix this.

Comment (YxAc) [reply common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java]: okya, i will take a closer look at the problem 1935 and try to fix, thx for your remind.

File: core/src/main/java/com/datastrato/gravitino/metrics/MetricsSystem.java [hunk 2]
```diff
@@ -33,7 +34,7 @@
 public class MetricsSystem implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(MetricsSystem.class);
   private final String name;
-  private final MetricRegistry metricRegistry;
+  @Getter private final MetricRegistry metricRegistry;
```
Comment (qqqttt123) [inline core/src/main/java/com/datastrato/gravitino/metrics/MetricsSystem.java:37]: I'm not sure that we should use `lombok` here. I know this is an optimization of IDEA. But maybe we should keep consistent with other places of the module `core`.

Comment (YxAc) [reply core/src/main/java/com/datastrato/gravitino/metrics/MetricsSystem.java:37]: yes, i think it is consistent with other places of the module `core` , such as `BaseMetalake` which using `lombok Getter`.

or i didn't get what you mean, can you give me more details? @qqqttt123

Comment (qqqttt123) [reply core/src/main/java/com/datastrato/gravitino/metrics/MetricsSystem.java:37]: 1. It's ok that we don't use `lomok`.  `BaseMetalake` is related to `serialization/deserialization`. It's ok to use `lomok`. 
2. I also want to reduce unnecessary code. I want to fix this issue using the least code.

Comment (YxAc) [reply core/src/main/java/com/datastrato/gravitino/metrics/MetricsSystem.java:37]: ok, got it. So is it ok by only removing @VisibleForTesting? @qqqttt123

Comment (YxAc) [reply core/src/main/java/com/datastrato/gravitino/metrics/MetricsSystem.java]: > ok, got it. So is it ok by only removing `@VisibleForTesting`? @qqqttt123

hi @qqqttt123 , i have submitted a new pull request #2191 . Can you take a look in your spare time? thx.

**Pull request closed without merging**